tests/integration/_common.sh

```diff
@@ -2,7 +2,7 @@
 
 if [ "xxx$root" = "xxx" ];
 then
-    root="$(mktemp -d)"
+    root="$(mktemp -d -t leitXXXX)"
```

Comment:
it's not obvious that this `-t leitXXXX` hack fixes Mac OS X. please put down a comment